auth/v1.js

```diff
@@ -57,10 +57,12 @@ var api = new API({
     table:      /^[A-Za-z][A-Za-z0-9]{2,62}$/,
 
     // Patterns for AWS
-    bucket:     /^[a-z0-9][a-z0-9-]{1,62}[a-z0-9]$/
+    bucket:     /^[a-z0-9][a-z0-9-]{1,62}[a-z0-9]$/,
                 // we could allow "." too, but S3 buckets with dot in the name
                 // doesn't work well with HTTPS and virtual-style hosting.
                 // Hence, we shouldn't encourage people to use them
+    // Project for sentry (and other per project resources)
+    project:    /^[a-zA-Z0-9_-]{1,22}$/,
```

Comment:
Nope, and maybe I should call it sentryProject instead
..
I think I'll use the same pattern for statsum projects though...

Note: it's evident when this shows up in API methods...

Note: if an scm project wanted sentry access, we could grant them a scope... But we would have to manually add their users to sentry, as admins or create a team for them and transfer sentry projects to that...

So this won't make sentry 100% self-service, but will make authentication easier to automate..